enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestionResult.tsx

```diff
@@ -28,6 +34,10 @@ interface InteractiveQuestionResultProps {
 export const InteractiveQuestionResult = ({
   height,
 }: InteractiveQuestionResultProps): ReactElement => {
+  const [isFilterOpen, setIsFilterOpen] = useState(false);
```

Comment:
I'm going to change this in a bit when I get some time to use a single `useState` with `filter | summarize | notebook`.